### PR TITLE
New: Add Catalan Language - Lang 39

### LINKED ISSUE #6257: Catalan language is not in the list
**Is your feature request related to a problem? Please describe.**
When I want to set Catalan language to a movie I discover that this language is not in the list

**Describe the solution you'd like**
"Catalan" language appear should be in the list

[AB#611](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/611)

### PR DESCRIPTION
**Pull request by dtalens**
#### Database Migration
NO

#### Description
Add Catalan language support. Translate complete #7415

#### Todos
- [ ] Tests
- [x] Translation Keys (./src/NzbDrone.Core/Localization/Core/ca.json)
- [ ] [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Fixes #6257

### CODE REVIEW

**Comment by dtalens** [comment]
Can someone help me debug code and figure out why some checks are failing? Thanks.

File: src/NzbDrone.Core.Test/ParserTests/LanguageParserFixture.cs [hunk 1]
```diff
@@ -367,7 +367,7 @@ public void should_parse_language_slovak(string postTitle)
             var result = Parser.Parser.ParseMovieTitle(postTitle);
             result.Languages.Should().BeEquivalentTo(Language.Slovak);
         }
-
+
```
Comment (Qstick) [inline src/NzbDrone.Core.Test/ParserTests/LanguageParserFixture.cs:370]: Get rid of this whitespace

File: src/NzbDrone.Core/Parser/LanguageParser.cs [hunk 2]
```diff
@@ -189,13 +190,19 @@ public static List<Language> ParseLanguages(string title)
             {
                 languages.Add(Language.Slovak);
             }
-
+
```
Comment (Qstick) [inline src/NzbDrone.Core/Parser/LanguageParser.cs:193]: Get rid of this whitespace

**Review by Qstick — CHANGES_REQUESTED** [review]

File: src/NzbDrone.Core.Test/ParserTests/LanguageParserFixture.cs (hunk 1 above)
Comment (dtalens) [reply src/NzbDrone.Core.Test/ParserTests/LanguageParserFixture.cs]: I've removed all whitespaces and blank lines, yet other checks still fail, thanks @Qstick

Comment (Qstick) [reply src/NzbDrone.Core.Test/ParserTests/LanguageParserFixture.cs]: Single blank lines are fine, and needed in some cases. Whitespace is not

**Comment by dtalens** [comment]
Hi @Qstick, when can this full request be merged? Thanks!

**Comment by dtalens** [comment]
Hi, when do you estimate that this pull request will be integrated? Thanks.

**Comment by dtalens** [comment]
Hello, this pull request has been open for months and now you close it without adding it. Also, you told me that Catalan would be language number 39 and now I see that you have added others. What am I supposed to do now?

**Pull request closed without merging**